### PR TITLE
Added changes to support loading msi's by id from symphony.yml

### PR DESCRIPTION
**Pull request by hattan**
Added changes to support loading msi's by id from symphony.yml

### CODE REVIEW

File: scripts/gitlab-server/gitlab-runner-setup.sh [hunk 1]
```diff
@@ -33,34 +34,38 @@ shArgs.arg "CERT_PATH" -c --cert-path PARAMETER true
...
     check_inputs
+
     check_az_is_logged_in
 
     if [ "$MODE_FULL" == true ]; then
         _debug "Running in full mode."
-        for i in {1..5}; do
-            local vmName="$VM_PREFIX-$i"
-            create_single_vm "$vmName"
+        for level in {1..5}; do
```
Comment (davesee) [inline scripts/gitlab-server/gitlab-runner-setup.sh:50]: Should we switch this out to 0..4 or is that irrelevant?

**Review by davesee — APPROVED** [review]
LGTM

File: scripts/gitlab-server/gitlab-runner-setup.sh (hunk 1 above)
Comment (hattan) [reply scripts/gitlab-server/gitlab-runner-setup.sh:50]: Good call, I will make the change.

File: scripts/gitlab-server/gitlab-runner-setup.sh [hunk 2]
```diff
@@ -90,15 +99,23 @@ add_server_private_ip_to_hosts_file() {
...
-    local msiId=$(az identity create -n $msiName -g $RESOURCE_GROUP | jq -r ".id")
-    echo $msiId
+get_msi_resource_id() {
+    local msiId=$1
+    local resourceId=""
+    resourceId=$(az identity list --query "[?clientId=='$msiId'].{id:id}" -o tsv)
+    echo $resourceId
 }
 
+find_msi_by_level () {
```
Comment (davesee) [inline scripts/gitlab-server/gitlab-runner-setup.sh:113]: find_msi_by_level () {
    local level="level$2"
    local msiId=""
    msiId=$(yq -r '.levels[] | select(.level == "'$level'").msiId' ../../.data/symphony.yml)
    echo $msiId
}

**Pull request merged**